### DISCUSSION TITLE
Wazuh white_list

### DISCUSSION THREAD

**Discussion by iamenderst** [Security-Onion-Solutions/securityonion#5176] [Unsupported Versions] [answered]
In Wazuh trying to whitelist an IP at the agent.
In ossec.conf I added below <ossec_config>
`<global>`
`  <white_list>xxx.xxx.xxx.xxx</white_list>`
`</global>`
...
 then restarted the agent. Still Wazuh bans the IP. That must be in the right place in the right file because Wazuh complained when I tried to restart the agent while working on it.

Security Onion Version 2.3.61

**Comment by gcsrellis** [comment]
There should be a location inside the .conf file with an ip or so already in it.  Add your IP in there instead of a new line.

**Reply by iamenderst** [reply]
If there is, it's not in the conf files I got.
Also tried adding white_list to the manager, no change.

`<ossec_config>`
`<global>`
`  <white_list>xxx.xxx.xxx.xxx</white_list>`
` </global>`
` `
`  <client>`
`    <server>`
...

**Comment by gcsrellis** [comment, marked as answer]
Are you trying to add this to the global.sls file on the master or the /opt/so/conf/wazuh/ossec.conf file on each node?  I've added IP's under the active response in ossec.conf and do not have any more issues.

**Reply by iamenderst** [reply]
Well there it is, /opt/so/conf/wazuh/ossec.conf
I've been working in /var/ossec/etc/ossec.conf on the agent and manager.
Only problem with that now, is the IP is white listed in all of the environment instead of one host.
